npm install @ng-select/ng-select@7.0.0 requiring angular 11

**Describe the bug**
npm install @ng-select/ng-select@7.0.0 requiring angular 11

**To Reproduce** 
Steps to reproduce the behavior (if example is not provided):
1. Create new angular project using ng cli v12 `ng new test` (using all default prompts)
2. `npm install @ng-select/ng-select@7.0.0`
3. See error

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: frontend@0.0.0
npm ERR! Found: @angular/common@12.0.4
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^12.0.4" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@">=11.0.0 <12.0.0" from @ng-select/ng-select@7.0.0
npm ERR! node_modules/@ng-select/ng-select
npm ERR!   @ng-select/ng-select@"^7.0.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Expected behavior**
ng-select version 7.0.0 installs in angular 12 project

**Desktop (please complete the following information):**
 - OS: Windows 10

```
 % ng --version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 12.0.4
Node: 14.16.1
Package Manager: npm 7.16.0
OS: win32 x64

Angular: 12.0.4
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1200.4
@angular-devkit/build-angular   12.0.4
@angular-devkit/core            12.0.4
@angular-devkit/schematics      12.0.4
@schematics/angular             12.0.4
rxjs                            6.6.7
typescript                      4.2.4
```

--- a/README.md
+++ b/README.md
@@ -15,6 +15,7 @@ See [Demo](https://ng-select.github.io/ng-select) page.
 
 | Angular| ng-select|
 | ------|:------:| 
+| >=12.0.0 <13.0.0 | v7.x |
 | >=11.0.0 <12.0.0 | v6.x |
 | >=10.0.0 <11.0.0 | v5.x |
 | >=9.0.0 <10.0.0 | v4.x |

--- a/src/ng-select/package.json
+++ b/src/ng-select/package.json
@@ -19,9 +19,9 @@
     "angular2"
   ],
   "peerDependencies": {
-    "@angular/common": ">=11.0.0 <12.0.0",
-    "@angular/core": ">=11.0.0 <12.0.0",
-    "@angular/forms": ">=11.0.0 <12.0.0"
+    "@angular/common": ">=12.0.0 <13.0.0",
+    "@angular/core": ">=12.0.0 <13.0.0",
+    "@angular/forms": ">=12.0.0 <13.0.0"
   },
   "dependencies": {
     "tslib": "^2.0.0"

